server/utils/passport/local-login.js

```diff
@@ -20,28 +24,24 @@ export default (req, res, next, user) => {
   })
   //If the user exists, serialize the user to the session
   req.login(user, function (err) {
+    if (err) return res.redirect(`${basePath}/login?e=${err}`)
+
     const { appDb } = req.app.locals
     const uid = user.uid
 
     appDb.collection(collections.configs).findOne(
       {
         owner: uid,
       },
-      function (err, configObj) {
-        if (err) {
-          console.error(err.message)
-        }
+      function (queryError, configObj) {
+        if (queryError) return res.redirect(`${basePath}/login?e=${queryError}`)
+
         if (!configObj) {
-          const defaultConfig = {
-            owner: uid,
-            config: defaultUserConfig,
-            name: 'default',
-            type: 'matrix',
-            readers: [uid],
-            created: new Date().toUTCString(),
-          }
-          appDb.collection(collections.configs).insertOne(defaultConfig)
+          appDb
+            .collection(collections.configs)
+            .insertOne(DEFAULT_CONFIG_AT_SIGNUP(uid))
```

Comment:
If we use a config model this might look like

```suggestion
          const config = ConfigModel.default({ owner: uid, readers: [uid] })
          appDb.collection(collections.configs).insertOne(config)
```

Reply:
[de48040](https://github.com/AMP-SCZ/dpdash/pull/293/commits/de48040cb5b39f723d4ad1877be815b064027992)